.env.example

```diff
@@ -12,4 +12,9 @@ SMTP_ADDRESS=
 SMTP_PORT=465
 SMTP_USERNAME=
 SMTP_PASSWORD=
-TLS=true
\ No newline at end of file
+TLS=true
+
+# Database Configuration
```

Comment:
Should we add in the README the default values in the .env?

Reply:
@pmareke it would be better if we add default values to .env.example

Reply:
✅ 🚀